From 004d4a402349b70f0cb0cc1c69074ab05d0e768c Mon Sep 17 00:00:00 2001
From: Boerge Struempfel <boerge.struempfel@gmail.com>
Date: Sat, 6 Jul 2024 21:07:48 +0200
Subject: [PATCH 1/3] kontron-sl-mx6ul: Reduce list of supported boot devices

The AutomationOne Gateway has neither eMMC nor is expected to support
booting from other sources (USB, PXE and DHCP) so we trimmed the list to:

 - MMC (SD card)
 - UBIFS

Upstream-Status: Inapropriate [configuration]

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
Signed-off-by: Boerge Struempfel <boerge.struempfel@gmail.com>
---
 include/configs/kontron-sl-mx6ul.h | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/include/configs/kontron-sl-mx6ul.h b/include/configs/kontron-sl-mx6ul.h
index 1c92cd78767..1f077ae06e3 100644
--- a/include/configs/kontron-sl-mx6ul.h
+++ b/include/configs/kontron-sl-mx6ul.h
@@ -31,12 +31,8 @@
 
 /* Boot order for distro boot */
 #define BOOT_TARGET_DEVICES(func) \
-	func(MMC, mmc, 1) \
 	func(MMC, mmc, 0) \
-	func(UBIFS, ubifs, 0, UBI, boot) \
-	func(USB, usb, 0) \
-	func(PXE, pxe, na) \
-	func(DHCP, dhcp, na)
+	func(UBIFS, ubifs, 0, UBI, boot)
 #include <config_distro_bootcmd.h>
 
 /* MMC Configs */
-- 
2.45.2

